core/client/app/controllers/setup/two.js

```diff
@@ -55,9 +55,14 @@ export default Ember.Controller.extend(ValidationEngine, {
                         }]
                     }
                 }).then(function () {
+                    // Don't call the success handler, otherwise we will be redirected to admin
+                    self.controllerFor('application').set('skipAuthSuccessHandler', true);
```

Comment:
`controllerFor` is going away.  Instead we should inject the application controller and then reference it like a normal property:

Above, where `notifications` and `ghostPaths` are injected:
`application: Ember.inject.controller()`

And then this can be changed to:
`self.get('application').set(...);`